sample_project/Source/sample_project/Feature Layer/FeatureLayer.cpp

```diff
@@ -38,51 +38,93 @@ void AFeatureLayer::OnResponseRecieved(FHttpRequestPtr Request, FHttpResponsePtr
 			//get the array field features
 			TArray<TSharedPtr<FJsonValue>> Features = ResponseObj->GetArrayField("features");
 
-			//parse through the features in order to get individual properties associated with the features
-			for (int i = 0; i != Features.Num(); i++)
-			{
-				//create a new feature object to store the data received associated with this feature iteration
-				FFeatureLayerProperties featureLayerProperties;
-				auto feature = Features[i]->AsObject();
-				//get the properties field associated with the individual feature
-				auto properties = feature->GetObjectField("properties");
 
-				/*outfield can be set in the scene on bp_feature
-				this loop will take each outfield set in the scene and check to see if the outfield exists
-				if it does exist, it will return the result of the outfield associated with this feature
-				if it does not exist, it will return and error message in the scene*/
-				for (auto outfield : WebLink.OutFields)
+			if(!bNewLink)
+			{
+				//parse through the features in order to get individual properties associated with the features
```

Comment:
Just a style nit, we probably should do `// Something something`, so space after comment and first work capitalize

```suggestion
				// Parse through the features in order to get individual properties associated with the features
```